public/data/Adebayoibrahim.json

```
@@ -0,0 +1,13 @@
+{
+ "name": "Abdulmumeen Ibrahim",
+ "location": "Nigeria",
+ "bio": "An Ambitious Web Programmer",
+ "avatar": "https://github.com/<Adebayoibrahim>.png",
```

```suggestion
 "avatar": "https://github.com/Adebayoibrahim.png",
```
Fix this also. Your avatar in not rendering because of this.